req_compile/repos/findlinks.py

```diff
@@ -14,21 +15,63 @@
 from req_compile.repos.repository import Candidate
 
 
+def _relativize(from_path: Path, to_path: Path) -> Path:
+    """Compute the relative path from one location to another.
+
+    Unlike `pathlib.Path.relative_to`, this handles the case where a path
```

Comment:
Doesn't os.path.relpath do this?  I can't believe you have to write this

Reply:
Oh wow, that's exactly what I want. I thought pathlib just used that but clearly not. I'll happily switch

Reply:
Updated!